libsplinter/src/admin/client/event/ws.rs

```diff
@@ -0,0 +1,439 @@
+// Copyright 2018-2021 Cargill Incorporated
```

Comment:
Consider moving this to client::event::ws::actix_web_1 module and guarding with the actix_web_1 feature.

Reply:
Moved this to `client::event::ws::actix_web_client`, to provide a similar separation. This being purely client code, and not relying on actix-web, it did not make sense to put in the actix_web_1 feature